cmake/iotjs.cmake

```diff
@@ -508,6 +508,14 @@ if (NOT BUILD_LIB_ONLY AND CREATE_SHARED_LIB)
     ${EXTERNAL_LIBS})
 endif()
 
+# Install headers
+if("${INCLUDE_INSTALL_DIR}" STREQUAL "")
```

Comment:
It seems better to put the relevant codes in one place. I suggest moving this code block to the line, 488.